pages/docs/concepts/asyncapi-document/variable-url.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Variables in URL
+weight: 235
+---
+
+Event-driven architecture benefits from a defined base URL and rules for URL variables. You can easily manage multiple API endpoints, handling various server configurations and environments.
+
+URL variables are placeholders for values you can replace during runtime. Async API enables you to construct dynamic URLs with query parameters and additional information while enhancing the flexibility and maintainability of your API specifications.
+
+This article assumes that you have a basic understanding of AsyncAPI. For more information, refer to the [Event-Driven Architectures](../../tutorials/getting-started/event-driven-architectures.md).
```

Comment:
That link 404s because it is appending the `/md` at the end, breaking the link. If you remove that, it should work.

**Broken link:**
https://deploy-preview-1890--shimmering-choux-eb0798.netlify.app/docs/tutorials/getting-started/event-driven-architectures.md 
<img width="300" alt="Screen Shot 2023-08-31 at 5 48 11 PM" src="https://github.com/asyncapi/website/assets/19964402/7138dc72-9c2b-45ae-8ca2-ed422469e8f2">